eng/Directory.Build.Data.props

```diff
@@ -35,6 +35,10 @@
     <AddDotnetfeedProjectSource>false</AddDotnetfeedProjectSource>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TF_BUILD)' == 'true'">
```

Comment:
I don't think TF_BUILD is ever set any where. We will need to figure out the better condition.

Reply:
TF_BUILD property is actually set as a [predefined variable for Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#system-variables)

Reply:
OK. Do we know what exactly setting this ContinuousIntegrationBuild option ends up doing? Or why it cannot be set all the time?

Reply:
The best I can tell[ from here ](DeterministicSourcePaths)is that `ContinuousIntegrationBuild` property informs the SDK that it is running on the CI server which is by default deterministic. The problem I was having earlier was that it also set `DeterministicSourcePaths` to true which caused projects that did not reference the SourceLink package to break because `SourceRoot` property was not set. However for all our Shipping Client Libraries this problem is not occurring since they reference the SourceLink package.

Reply:
Do we know if this make us green from the nuget package manager standpoint? I believe that is our primary reason for trying to make this change.

Reply:
Yes, I tested and it does make us green.